robot-server/tests/integration/protocols/deck_coordinate_load_fail.py

```diff
@@ -0,0 +1,23 @@
+metadata = {
+    "protocolName": "Deck Coordinate PAPIv2 Test",
+}
+
+requirements = {"robotType": "Flex", "apiLevel": "2.15"}
+
+
+def run(context):
+    pipette = context.load_instrument("p1000_single_gen2", mount="left")
+
+    labware = context.load_labware("armadillo_96_wellplate_200ul_pcr_full_skirt", "d3")
+    assert labware.parent == "D3"
+
+    thermocycler = context.load_module("thermocyclerModuleV2")
+    assert thermocycler.parent == "B1"
+
+    module = context.load_module("temperatureModuleV2", "A1")
+    assert module.parent == "A1"
+
+    pipette.move_to(labware.wells()[0].top())
+    module_labware = module.load_labware("armadillo_96_wellplate_200ul_pcr_full_skirt")
+
+    pipette.move_to(module_labware.wells()[0].top())
```

Comment:
Since we're expecting this to fail on this `context.load_module()`, can we trim out the parts of this file that won't run?

```suggestion
    # This should fail because the Thermocycler is in the way.
    module = context.load_module("temperatureModuleV2", "A1")
```